src/analytics/management/commands/export_items_to_personalize.py

```diff
@@ -0,0 +1,195 @@
+import os
+import time
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+
+from analytics.utils.analytics_file_utils import (
+    export_data_to_csv_in_chunks,
+    read_last_processed_ids,
+    remove_file,
+)
+from analytics.utils.analytics_mapping_utils import build_doc_props_for_item
+from researchhub_comment.related_models.rh_comment_model import RhCommentModel
+from researchhub_document.models import ResearchhubUnifiedDocument
+
+OUTPUT_FILE = "./exported_items.csv"
+TEMP_PROGRESS_FILE = "./item-export-progress.temp.json"
+EXPORT_FILE_HEADERS = [
+    "ITEM_ID",
+    "CREATION_TIMESTAMP",
+    "item_type",
+    "item_subtype",
+    "internal_item_id",
+    "unified_document_id",
+    "created_by_user_id",
+    "bounty_id",
+    "bounty_amount",
+    "bounty_type",
+    "bounty_expiration_timestamp",
+    "title",
+    "journal",
+    "pdf_license",
+    "oa_status",
+    "twitter_score",
+    "slug",
+    "authors",
+    "updated_timestamp",
+    "publication_timestamp",
+    "publication_year",
+    "hubs",
+    "hub_slugs",
+    "primary_hub",
+]
+MODELS_TO_EXPORT = [
+    "ResearchhubUnifiedDocument",
+    "RhCommentModel",
+]
+
+
+def map_document_data(docs):
+    data = []
+    for doc in docs:
+        try:
+            record = {}
+            item_type = doc.get_client_doc_type()
+            specific_doc = doc.get_document()  # paper, post, ...
+
+            doc_props = build_doc_props_for_item(doc)
+            record = {**doc_props}
+            record["ITEM_ID"] = item_type + "_" + str(specific_doc.id)
+            record["item_type"] = item_type
+            record["internal_item_id"] = str(specific_doc.id)
+            record["CREATION_TIMESTAMP"] = int(
+                time.mktime(doc.created_date.timetuple())
+            )
+            record["updated_timestamp"] = int(time.mktime(doc.updated_date.timetuple()))
+
+            if specific_doc.created_by:
+                record["created_by_user_id"] = str(specific_doc.created_by.id)
+
+            data.append(record)
+        except Exception as e:
+            print("Failed to export doc: " + str(doc.id), e)
+
+    return data
+
+
+def map_comment_data(comments):
```

Comment:
why don't we store the comment content here? is it because it gets too long?

Reply:
According to documentation, it is not necessary. The important part are categorical information and user interaction data (how people engage with the comment)